### ISSUE TITLE
False positive E1029 on API GW Stage Variables with dot syntax

### ISSUE THREAD

**Issue by alytle** [aws-cloudformation/cfn-python-lint#1484]
*cfn-lint version: 0.29.6*

In #511 you fixed a false positive when referencing API GW Stage Variables via the syntax `${stageVariables['variableName']}`, but they can also be referenced using dot syntax: `${stageVariables.variableName}`.

```
  Request:
    Type: AWS::ApiGateway::Method
    Properties:
      Integration:
        Type: HTTP_PROXY
        IntegrationHttpMethod: !Ref HttpMethod
        ConnectionType: VPC_LINK
        ConnectionId: "${stageVariables.VPCLink}"  <--- E1029
```
